themes/default/content/blog/nodejs-binaries-with-pkg/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Node.js Native Binary Compilation Using vercel/pkg"
+date: 2022-09-13
+meta_desc: "Using Vercel pkg to create native binaries for nodejs providers"
+
+#meta_image: infra-observability.png
+authors:
+- daniel-bradley
+tags:
+- engineering
+- nodejs
+- publishing
+- providers
+
+meta_image: "meta.png"
+---
+
+In Pulumi's engineering department we build and distribute our tools as native binaries as there's no local dependencies needed. Go provides a great story for building self-contained binaries that target modern operating systems.
```

Comment:
```suggestion
In Pulumi's engineering department, we often build and distribute tools as native binaries to avoid the need for additional dependencies on user machines. Most of these tools are written in Go, which has good support for building self-contained binaries that target modern operating systems.
```